src/ATDPApplication.ts

```diff
@@ -112,30 +118,28 @@ export abstract class ATDPApplication<T> extends ACLUEWrapper {
 
     const i18nPromise = I18nextManager.getInstance().initI18n();
 
-    Promise.all([configPromise, i18nPromise]).then(() => {
-      this.tourManager = new TourManager({
-        doc: document,
-        header: () => this.header,
-        app: () => this.app
-      });
-
-      BaseUtils.mixin(this.options, {
-        showHelpLink: this.tourManager.hasTours() ? '#' : false // use help button for tours
+    Promise.all([configPromise, i18nPromise])
+      .then(() => {
+        this.build(document.body, {replaceBody: false});
+      })
+      .then(() => {
```

Comment:
Why exactly the `then` chaining?
Wouldn't it be more readable if we convert this whole thing into an inline async function such that we can actually await things?

```
(async () => {
  await Promise.all([configPromise, i18nPromise]);
  
  this.build(document.body, {replaceBody: false});

  this.tourManager = new TourManager({
  ...
})();
```